### PR TITLE
update dcl/schemas

### PR DESCRIPTION
**Pull request by menduz**
# What? 
...

# Why? 
...

### CODE REVIEW

**Review by 2fd — CHANGES_REQUESTED** [review]
@menduz

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
   },
   "dependencies": {
     "@dcl/catalyst-peer": "^1.0.14",
-    "@dcl/crypto": "^3.2.3",
+    "@dcl/crypto": "^3.3.1",
```
[the remark this answers is not in the document]
Comment (menduz) [reply package.json:61]: it is dcl/crypto, not schemas

File: scripts/fetchSceneContents.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@
 import { fetch } from 'undici'
 import { CatalystClient } from 'dcl-catalyst-client/dist/CatalystClient'
 import * as fs from 'fs'
-import { ContentMapping, EntityType } from '@dcl/schemas'
+import { ContentMapping } from '@dcl/schemas'
```
[the remark this answers is not in the document]
Comment (menduz) [reply scripts/fetchSceneContents.ts:13]: this is an auxiliary script

**Comment by 2fd** [comment]
❤️

**Pull request merged**